scripts/classes/extensions-installer.js

```diff
@@ -95,13 +102,17 @@ class ExtensionsInstaller {
     const install = this.localExtensions.length ? npmInstall : yarnInstall;
 
     const installedExtensions = [...this.localExtensions, ...this.extensionsToInstall];
-    excludePackages.forEach(packageName => delete packageJsonTemplate.dependencies[packageName]);
+    _.forEach(excludePackages, packageName => delete packageJsonTemplate.dependencies[packageName]);
```

Comment:
This function both excludes packages and installs them. Can you extract the logic for excluding packages from this function?